app/models/transcript.rb

```diff
@@ -10,7 +10,7 @@ class Transcript < ApplicationRecord
 
   validates :original_url, presence: true
 
-  validates :mime_type, inclusion: {in: %w[text/plain application/json application/pdf]}, allow_blank: true
+  validates :mime_type, inclusion: {in: %w[text/plain application/json]}, if: :status_complete?, allow_blank: true
```

Comment:
May be worth uploading the [4 example files](https://github.com/Podcastindex-org/podcast-namespace/tree/main/transcripts) in the podcastindex transcript section, to make sure they pass this.  (And to see what mime-types Porter is returning for them).